vars/tagGit.groovy

```diff
@@ -32,6 +32,8 @@ def call(Map <String, ?> config = [:]) {
         node(POD_LABEL) {
           container('git') {
 
+              sh "ssh-keyscan -t ${ssh_key_type} ${git_server} >> ~/.ssh/known_hosts"
```

Comment:
I think these need to be prefixed with the hasmap var name. 

Also, because I intended these to be optional, keeping the `if` guard makes sense.

```suggestion
              sh "ssh-keyscan -t ${config.ssh_key_type} ${config.git_server} >> ~/.ssh/known_hosts"
```